### PR TITLE
Update dns-horizontal-autoscaling.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by xichengliudui** [comment]
/cc @tengqm 
Ref: https://github.com/kubernetes/website/pull/12948

File: content/zh/docs/tasks/dns-horizontal-autoscaling.md [hunk 1]
```diff
@@ -427,7 +427,7 @@ the autoscaler Pod.
 patterns: *linear* and *ladder*.
 -->
 
-* 集群比例自动伸缩器应用程序与 DNS 服务分开部署。
+* 集群比例自动伸缩器应用程序与 DNS 服务分开 deployment。
```
Comment (idealhack) [inline content/zh/docs/tasks/dns-horizontal-autoscaling.md]: revert this

**Comment by xichengliudui** [comment]
已修改。

File: content/zh/docs/tasks/dns-horizontal-autoscaling.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Kubernetes cluster.
 <!--
 List the Deployments in your cluster in the kube-system namespace:
 -->
-在 kube-system 命名空间中列出集群中的部署：
+在 kube-system 命名空间中列出集群中的 Deployment：
```
Comment (stewart-yu) [inline content/zh/docs/tasks/dns-horizontal-autoscaling.md]: 修改为：

列出集群中命名空间kube-system下的负载

是不是更合理、通顺点？下面几处请类似修改

Comment (xichengliudui) [reply content/zh/docs/tasks/dns-horizontal-autoscaling.md]: 行。

**Pull request merged**